src/Util/cloudinaryProps.js

```diff
@@ -0,0 +1,18 @@
+import {Util} from "cloudinary-core";
+
+/**
+ * Returns a camelCase version of given props, filtered using cloudinaryOptions
```

Comment:
@maoznir , don't you find the need to run on this list twice, once to get the cloudinary props and once to get the non-cloudinary props a bit too much? 
Can't we have a single run that separates the cloudinary from non-cloudinary props?